internal/stores/contracts.go

```diff
@@ -76,9 +83,36 @@ func (c dbContract) convert() (bus.Contract, error) {
 		ContractMetadata: bus.ContractMetadata{
 			RenewedFrom: c.RenewedFrom,
 			TotalCost:   types.NewCurrency(c.TotalCost),
-			Spending:    bus.ContractSpending{}, // TODO
+			Spending: bus.ContractSpending{
+				Uploads:     types.NewCurrency(c.UploadSpending),
+				Downloads:   types.NewCurrency(c.DownloadSpending),
+				FundAccount: types.NewCurrency(c.FundAccountSpending),
+			},
 		},
-	}, nil
+	}
+}
+
+// convert converts a dbContract to a bus.ArchivedContract.
+func (c dbArchivedContract) convert() bus.ArchivedContract {
+	return bus.ArchivedContract{
+		ID:        c.FCID,
+		HostKey:   c.Host,
+		RenewedTo: c.RenewedTo,
+
+		Spending: bus.ContractSpending{
+			Uploads:     types.NewCurrency(c.UploadSpending),
+			Downloads:   types.NewCurrency(c.DownloadSpending),
+			FundAccount: types.NewCurrency(c.FundAccountSpending),
+		},
+	}
+}
+
+func gobFCID(fcid types.FileContractID) ([]byte, error) {
```

Comment:
I added `gobEncode` once to the `contracts_test` file, we should also remove that in favour of using `gobFCID` (although I like `gobEncode` better). I would also panic instead of returning an error.

Reply:
I actually updated `gobEncode` to use `gobFCID` but I'll rename it and remove the test helper completely.